### ISSUE TITLE
PHP 7 Support

### ISSUE THREAD

**Issue by DrSkyfaR** [wumix/Smart-Space#27]
Hey ist für deine Adaption ein Anpassung auf PHP 7 von dir geplant ?

**Comment by wumix** [comment]
Normal schon

ApoRes1337 <notifications@github.com> schrieb am Fr., 9. Juni 2017, 03:11:

**Comment by DrSkyfaR** [comment]
Das ist super :) 
Ich konnte leider deinem Forum nichts entnehmen dazu
weil die Aktivierung ja nicht funktioniert. Vielleicht schaust du ja nochmal danach.

**Comment by wumix** [comment]
Die Aktivierung geht nicht?

ApoRes1337 <notifications@github.com> schrieb am Fr., 9. Juni 2017, 11:54:

**Comment by DrSkyfaR** [comment]
Leider nein auch nach mehrmaligen zusenden der Aktivierungskeys kam nichts durch.

**Comment by ViperAix** [comment]
Gibt es was neues bezüglich Kompatiblität mit PHP 7 ?

Bekomme leider diese Meldung, wenn ich versuche es mit PHP 7.1.1 laufen zu lassen: http://prntscr.com/g7wa9q

**Comment by DrSkyfaR** [comment]
Nein, ich denke da wird auch nichts mehr kommen das Projekt scheint so gut wie eingestellt zu sein.

**Comment by ViperAix** [comment]
Das ist doof - schade...

**Comment by ViperAix** [comment]
Kurze Frage: Könnte man das denn selber anpassen, oder ist Smarty mit PHP 7.1.x gar nicht mehr kompatibel?

Ich hätte schon Interesse daran das ganze Projekt auf einen aktuellen Stand zu bringen.

**Comment by DrSkyfaR** [comment]
Grundsätzlich gäbe es die Möglichkeit du müsstest aber erstmal das veraltete Smarty Template auf Smarte 3.X aktualisieren damit es wieder PHP 7.1 kompatibel wird. 

Es ist nobel das aktualisieren zu wollen, ich denke du hast aber weniger Arbeit wenn du dir gleich dein eigenes Design ausarbeitest. Zu mal eine Anpassung an 7.1 nur bedingt was bringt, Bugs und Fehler werden dann ja dennoch nicht behoben da Wumix so wie es aussieht nicht mehr all zu viel daran machen wird, wenn überhaupt.

**Comment by ViperAix** [comment]
Hm okay...
Ich überlege mal wie ich da weiter vorgehen werde. Danke Dir schonmal! :)

**Comment by DrSkyfaR** [comment]
Schreib dir am besten auf dem Core vom aktuellen 2Moons ein eigenes Responsive Design, da sparste dir dann vor allem das umständliche Smarty Templating. Wenn du vor hast ernsthaft nochmal was damit anzufangen setze auf die Möglichkeiten von PHP 7.1, HTML 5 und CSS3 bzw dazugehörigen Javascript. Damit fährst du langfristig definitiv am besten.

Ich wünsche dir aber weiterhin viel Spaß und Erfolg.
Ps. Solltest du da irgendwas mal anfangen würde ich mir gerne die Fortschritte dazu mal anschauen. :)

**Comment by reverence87** [comment]
So richtig Lust habe ich auch nicht mehr, an diesem Projekt weiter zu arbeiten. Aber vielleicht kommt das ja mal wieder ;-)

**Comment by ViperAix** [comment]
Bestimmt :-)

Ich werde heute mal anfangen auf dem Core vom aktuellen 2Moons ein eigenes Responsive Design zu basteln. Mal gucken was ich damit so hinbekommen werde...

**Comment by xzessmedia** [comment]
@dhavenith gibts nach mehr als einem Jahr dazu ein Ergebnis?

**Comment by wumix** [comment]
Hey durch Zuwachs und Hauskauf hatte ich so gut wie keine Zeit mehr.

Ich arbeite gerade an eine komplett aktuelle Version des Spiels &  des templates.

Da ich aber auch gerade viel Zeit in 2moons CE stecke wird mein Projekt erst danach fort gesetzt.
